website/docs/r/cos_bucket.html.markdown

```diff
@@ -165,6 +165,7 @@ resource "ibm_cos_bucket" "retention_cos" {
   resource_instance_id = ibm_resource_instance.cos_instance.id
   region_location      = "jp-tok"
   storage_class        = standard
+  hard_quota           = var.quota
```

Comment:
instead of var.quota can you provide some value

Reply:
done with the chnages